libs/blocks/media/media.css

```diff
@@ -103,13 +106,93 @@ div[class*="-up"] .media .foreground > .media-row {
   padding: 0 0 var(--spacing-m);
 }
 
+.media .default-list {
+  padding-left: 20px;
+  margin-top: var(--spacing-xs);
+  margin-bottom: var(--spacing-xs);
+}
+
+.media li::before {
+  content: "";
+  display: inline-block;
+  margin-left: 3px;
+}
+
+.media.checklist li {
+  background: url("/libs/img/ui/checkmark-green.svg") no-repeat 5px 5px transparent;
+  list-style-type: none;
+  margin-left: -25px;
+  padding: 0 0 1px 29px;
+  vertical-align: middle;
+}
+
+.media .subcopy {
+  font-size: var(--type-heading-xs-size);
+  line-height: var(--type-body-s-lh);
+  margin-top: var(--spacing-xs);
+  color: #2C2C2C;
+}
+
+.media h3.heading-xs {
+  margin-top: var(--spacing-s);
+  margin-bottom: var(--spacing-xs);
+}
+
+.media p.icon-area {
+  margin-bottom: 0;
+}
+
+.media p.subcopy-link {
+  margin-top: 0;
+  margin-bottom: 8px;
+}
+
+.media a.subcopy-link {
```

Comment:
We should always use the global defaults when possible. aka link colors and typography.css classes. You can remove this style all together and add a type class in the js. essentially targeting `.media .subcopy-link a.body-xxs` etc. LMK if you have questions.